ledger/sandbox-classic/BUILD.bazel

```diff
@@ -440,13 +440,5 @@ server_conformance_test(
     lf_versions = ["stable"],
     server_args = ["--contract-id-seeding=testing-weak"],
     servers = SERVERS,
-    test_tool_args = [
-        "--include=" + ",".join([
-            "DeeplyNestedValueIT:Accept",
-            "DeeplyNestedValueIT:RejectCreateCommand",
-            "DeeplyNestedValueIT:RejectExerciseCommand",
-            "DeeplyNestedValueIT:RejectCreateArgumentInCreateAndExerciseCommand",
-            "DeeplyNestedValueIT:RejectChoiceArgumentInCreateAndExerciseCommand",
-        ]),
-    ],
+    test_tool_args = ["--include=DeeplyNestedValueIT:Accept"],
```

Comment:
Why did these change?

Reply:
I should have remove also `:Accept`. All the test passed for sandbox classic.